.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+* Include screenshots and animated GIFs which help you demonstrate the steps to reproduce issue.
+* Explain why this enhancement would be useful to most Moira users.
+* Specify which version of Moira you're using.
+
+## Contributing a Code
+
+If you want to contribute a code you should choose an issue and comment that you want to work on it. If you cannot find 
+an issue for your contribution please fill an issue first. Please **DO NOT** work on code before you file an issue and comment
+about your wish to work on it.
+
+Here you can find issues that are good for first time contributors.
+
+* [Good First Issue](https://github.com/moira-alert/moira/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+* [Help Wanted](https://github.com/moira-alert/moira/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
+
+During implementation and submission of code follow style guides listed below.
+
+## Style Guides
+
+### Git Style Guide
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) philosophy to submit the code.
+If you are not familiar with Conventional Commits it's highly recommended to read 
+[documentation](https://www.conventionalcommits.org/en/v1.0.0/) about it.
+
+In general the rules for commiting are following:
+
+* Commit message should have three blocks: title, body and footer divided with empty lines.
+* At least title should be specified.
+* Start first line with type of commit. You can find commit types below.
+* Place summary in the first line.
+* If applicable specify the scope of commit.
+* In body describe the changes you did. Write mostly **why** you did changes instead **what** you changed.
+* You can split a body to multiple paragraphs using blank lines.
+* In footer place an additional information about your commit. Specify issue number in footer with `Relates` or `Closes` label.
+
+Structure of commit:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+We are using following commit types in our repo:
+
+* **build:** Changes that affect the build system or external dependencies
+* **ci:** Changes to our CI configuration files and scripts
+* **docs:** Documentation only changes
+* **feat:** A new feature
+* **fix:** A bug fix
+* **perf:** A code change that improves performance
+* **refactor:** A code change that neither fixes a bug nor adds a feature
+* **style:** Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+* **test:** Adding missing tests or correcting existing tests
+
+### Go Code Style Guide
+
+We are using [GolangCI-Lint](https://github.com/golangci/golangci-lint) to check our code for following the code style.
+Check your code with `make lint` before you will submit it. Code with broken style will not pass the CI and will not be
+reviewed. In future core contributors team can change the rules for linter in project.
```

Comment:
```suggestion
reviewed.
```

Of course we can change anything anytime. I don't see the point of reiterating this thought here. We don't strive to intimidate our reader, do we?